src/Value/Resolver.php

```diff
@@ -11,11 +11,13 @@ class Resolver implements ResolverInterface
      *
      * @param  mixed  $model
      * @param  string  $attribute
-     * @param  Illuminate\Support\Collection  $groups
+     * @param  \Illuminate\Support\Collection  $value
      * @return string
      */
-    public function set($model, $attribute, $groups)
+    public function set($model, $attribute, $value)
     {
+        $groups = $value; // to avoid Named Arguments issue but keep using better var name;
+
```

Comment:
I'm not a big fan of these changes. Why would `$value` be a better var name than `$groups` when using named arguments? (real question)

Reply:
Because `value` is reserved word.
`$this->set(value: $value, model: $model, attribute: $attr);` would fail.

Reply:
@toonvandenbos 
agree, sorry for not mentioning this. This is because the implemented [interface has this name](https://github.com/whitecube/nova-flexible-content/blob/7a5f9276e89c990f259852021f13811b8c59328a/src/Value/ResolverInterface.php#L7 ) plus I couldn't find `@no-named-arguments` tags or other docs/promises of supporting/not supporting named arguments introduced in PHP 8

Reply:
PS: I'm happy to revert these changes if it will help to merge this PR

Reply:
And there is [uninformative-words](https://github.com/szepeviktor/uninformative-words).
But it may not spark joy!

Reply:
Thank you for the insights. These arguments have indeed not been named consistently.

IMO the "clean" solution would be to rename [`$value` into `$groups` in `Value/ResolverInterface.php`](https://github.com/whitecube/nova-flexible-content/blob/7a5f9276e89c990f259852021f13811b8c59328a/src/Value/ResolverInterface.php#L7). This change should not break any existing implementations, plus we could avoid the uninformative `$value` variable (and more importantly that ugly `$groups = $value` statement which is honestly hurting my soul 😅).

Reply:
@lptn you can still edit this PR. I'll wait for your changes and give it a final review before merging. Thanks!

Reply:
@toonvandenbos 
done: renamed, also added some PHPDoc to the interface + updated a stub + extra cleanup (safe one)

Reply:
@lptn but forgot to commit

Reply:
@toonvandenbos 
Please review again when you have a time - the PR is ready. Thanks!